higher_health/views.py

```diff
@@ -20,6 +20,17 @@ def form_valid(self, form):
         self.request.session["triage_id"] = str(triage.id)
         return super().form_valid(form)
 
+    def get_context_data(self, **kwargs):
+        ctx = super(HealthCheckQuestionnaireView, self).get_context_data(**kwargs)
+        ctx["campuses"] = {
+            str(c.pk): str(c.university.pk)
+            for c in Campus.objects.filter().select_related("university")
```

Comment:
Since we only need the ID of the university, we don't actually need to do any joins here.

We can probably also only pull the fields that we need, since we know exactly what fields we want in advance.

So this could simplify to:
```
ctx["campuses"] = dict(Campus.objects.values_list("id", "university_id"))
```
And same with the universities:
```
ctx["universities"] = dict(University.objects.values_list("id", "province"))
```